<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CWorldEditorWindow</class>
 <widget class="QMainWindow" name="CWorldEditorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Prime World Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="CentralGLWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuDynamic_lighting">
     <property name="title">
      <string>Dynamic lighting</string>
     </property>
     <addaction name="actionLightingNone"/>
     <addaction name="actionLightingBasic"/>
     <addaction name="actionLightingWorld"/>
    </widget>
    <widget class="QMenu" name="menuBloom">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Bloom</string>
     </property>
     <addaction name="actionDisableBloom"/>
     <addaction name="actionEnableBloom"/>
     <addaction name="actionShowBloomMaps"/>
    </widget>
    <addaction name="actionReset_camera"/>
    <addaction name="actionBackface_culling"/>
    <addaction name="separator"/>
    <addaction name="actionWorld"/>
    <addaction name="actionCollision"/>
    <addaction name="actionObjects"/>
    <addaction name="actionMaterial_Animations"/>
    <addaction name="actionLights"/>
    <addaction name="actionOccluder_meshes"/>
    <addaction name="actionSky"/>
    <addaction name="separator"/>
    <addaction name="menuDynamic_lighting"/>
    <addaction name="menuBloom"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QDockWidget" name="ObjectPropertiesDock">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Object Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>272</width>
          <height>330</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QGroupBox" name="ConnectionsGroup">
           <property name="title">
            <string>Connections</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3"/>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="PropertiesGroup">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Properties</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="InstanceViewDock">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Instance View</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_6">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QTreeWidget" name="InstanceViewTreeWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string>Instance</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Type</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpen_CMDL">
   <property name="text">
    <string>Open CMDL</string>
   </property>
  </action>
  <action name="actionReset_camera">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Reset camera</string>
   </property>
  </action>
  <action name="actionBackface_culling">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Backface culling</string>
   </property>
  </action>
  <action name="actionWorld">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>World</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </action>
  <action name="actionCollision">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Collision</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="actionObjects">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Objects</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </action>
  <action name="actionPaths">
   <property name="text">
    <string>Paths</string>
   </property>
  </action>
  <action name="actionLightmaps">
   <property name="text">
    <string>Lightmaps</string>
   </property>
  </action>
  <action name="actionMaterial_Editor">
   <property name="text">
    <string>Material Editor</string>
   </property>
  </action>
  <action name="actionExtract">
   <property name="text">
    <string>Extract</string>
   </property>
  </action>
  <action name="actionRepack">
   <property name="text">
    <string>Repack</string>
   </property>
  </action>
  <action name="actionDump_file_list">
   <property name="text">
    <string>Dump file list</string>
   </property>
  </action>
  <action name="actionConvert_TXTR_to_DDS">
   <property name="text">
    <string>Convert TXTR to DDS</string>
   </property>
  </action>
  <action name="actionConvert_DDS_to_TXTR">
   <property name="text">
    <string>Convert DDS to TXTR</string>
   </property>
  </action>
  <action name="actionClose_all">
   <property name="text">
    <string>Close all</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionOpen_MREA">
   <property name="text">
    <string>Open MREA</string>
   </property>
  </action>
  <action name="actionRead_MREA">
   <property name="text">
    <string>Read MREA</string>
   </property>
  </action>
  <action name="actionMaterial_Animations">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>UV animations</string>
   </property>
   <property name="shortcut">
    <string>4</string>
   </property>
  </action>
  <action name="actionLights">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Lights</string>
   </property>
   <property name="shortcut">
    <string>5</string>
   </property>
  </action>
  <action name="actionLightingNone">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>None</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionLightingBasic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Basic</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionLightingWorld">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>World lights</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionSky">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sky</string>
   </property>
   <property name="shortcut">
    <string>7</string>
   </property>
  </action>
  <action name="actionOccluder_meshes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Occluder meshes</string>
   </property>
   <property name="shortcut">
    <string>6</string>
   </property>
  </action>
  <action name="actionDisableBloom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </action>
  <action name="actionEnableBloom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Bloom</string>
   </property>
  </action>
  <action name="actionShowBloomMaps">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Bloom Maps</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
